sale_service_project/README.rst

```diff
@@ -0,0 +1,72 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+====================
+Sale Service Project
+====================
+
+This module adds the option to assign materials and/or work to a service
+product type which serve as a template transferring this information to the
+line order which may be changed.
+This module extends the functionality of the *sale_service* module creating a
+new project for each order adding materials and works informed on the order
+line to the generated task.
+
+Usage
+=====
+
+To use this module, you need to:
+
+1.Create a service product and check 'Create Task Automatically'.
+2.Now you can add task works and materials for this product.
+3.You can compute the total price based in all materials and task works
+  assigned.
+4.Create sale order and assign an analytic account which will be parent of new
+  project.
+  If *invoice_on_timesheets* option is checked on analiytic account this sale
+  order will be invoiced from analytic lines. Otherwise sale order will be
+  invoiced normally.
+5.When you confirm the sale order odoo will create a project and tasks.
```

Comment:
Please give [proper formatting](http://docutils.sourceforge.net/docs/user/rst/quickref.html#enumerated-lists) to lists, or they appear broken.